### DISCUSSION TITLE
Run lifecycle commands in parallel

### DISCUSSION THREAD

**Discussion by sgurlt** [community/community#11813] [Codespaces]
Hi,

I am currently checking out if we are going to move from gitpod to codespaces.
Gitpod provides the possibility to run startup / lifecycle commands in parallel and do have a "wait" feature which allows user to tell "this should start when that is finished". Is something similar available for codespaces? 

Best regards,
Sebastian

**Comment by r3dpointsp** [comment]
in our use case we installed GNU parallel in the image and ran multiple bash functions in parallel within the one lifecycle script that CodeSpaces invokes, which works well enough for us. YMMV
